### ISSUE TITLE
Lambda Server Too many logins

### ISSUE THREAD

**Issue by MSH-11** [mikeizbicki/cmc-csci143#443]
When I try connecting to the lambda server I get this:
```
> ssh Mohamed.Hamouda.24@lambda.compute.cmc.edu -p 5055
Mohamed.Hamouda.24@lambda.compute.cmc.edu's password:
Too many logins for 'Mohamed.Hamouda.24'.
Last login: Wed Feb 14 17:28:56 2024 from 134.173.187.153
Connection to lambda.compute.cmc.edu closed.
```
I am sure I am not connected. I even restarted my device and still the same issue.

Is there a way to disconnect my current connections from your end @mikeizbicki ?

Thank you,

**Comment by mikeizbicki** [comment]
It looks like you're using tmux to manage sessions?  Tmux keeps live sessions running after disconnect, and so probably what happened is you had a number of sessions still going that you had forgotten about from the past.  The lambda server only allows students 8 concurrent connections at a time.

I tried to fix the problem for you by killing all of your bash and tmux processes that were still running.  (Most of them were from Feb 1.)  Specifically, I ran the incantation
```
$ ps -ef | grep Mohame | grep -E '(bash|tmux)' | tr -s ' ' | cut -f2 -d' ' | sudo xargs kill
```
Let me know if that worked.
